myst_nb/converter.py

```diff
@@ -248,6 +258,19 @@ def _flush_markdown(start_line, token, md_metadata):
         if token.type == "fence" and token.info.startswith(code_directive):
             _flush_markdown(md_start_line, token, md_metadata)
             options, body_lines = read_fenced_cell(token, len(notebook.cells), "Code")
+            # Parse :file or file: tags and populate body with contents of file
+            if "file" in options:
+                fl = re.search(r"( .*?\.[\w:]+)", token.content).group(0).lstrip()
+                flpath = Path(context.srcdir + "/" + fl).resolve()
+                if len(body_lines):
+                    LOGGER.warning(
+                        ("content of code-cell is being overwritten by `file`"),
+                        location=(context.srcdir + "/" + context.docname, token.map),
```

Comment:
passing `token.map` to location is wrong

Reply:
@AakashGfude can you review this comment and update. Thanks.

Reply:
hmm. I was trying to get the line numbers for this logger. Could not find another variable that can pinpoint that. @chrisjsewell why is it wrong though? should I just get one entry(the start line) from `token.map` and pass it?

Reply:
thanks @AakashGfude I think adding line number information is a big help to the `warning`.

Reply:
https://github.com/executablebooks/MyST-Parser/blob/231971502221d821839c68561ae6293ea8e5382e/myst_parser/docutils_renderer.py#L76